packages/block-library/src/comment-date/index.php

```diff
@@ -24,6 +24,9 @@ function render_block_core_comment_date( $attributes, $content, $block ) {
 	}
 
 	$classes = '';
+	if ( isset( $attributes['style']['elements']['link']['color']['text'] ) ) {
+		$classes .= ' has-link-color';
+	}
```

Comment:
```suggestion
	$classes = ( isset( $attributes['style']['elements']['link']['color']['text'] ) ) ? 'has-link-color' : '';
```
Sort of a nitpick: Since `$classes` is an empty string above, this adds an empty space before the classname. From what I can see, the `get_block_wrapper_attributes()` doesn't run `trim()` on these, so we'd end-up with unnecessary spaces before the class-names etc.
The same thing goes for a lot of other changes here so I'm not going to add a comment on each one of them, but these should probably be fixed.

Reply:
Which is preferred, with or without array_merge?
Without: https://github.com/WordPress/gutenberg/blob/trunk/packages/block-library/src/latest-comments/index.php#L120
With: https://github.com/WordPress/gutenberg/pull/47357/files#diff-a7a3de558cf559d59c7da7d387a02cee173cc31c5ea349077acae909b9f66b53R39

Reply:
Definitely without `array_merge()`... Array operations like that are slow.